pom.xml

```diff
@@ -56,9 +58,9 @@
         <sortpom-maven-plugin.version>2.8.0</sortpom-maven-plugin.version>
         <spring-beans.version>5.3.21</spring-beans.version>
```

Comment:
Try to remove explicit spring beans and core dependencies and see build succeeds.

Reply:
Done